qa/L0_server_status/server_status_test.py

```diff
@@ -131,57 +151,94 @@ def test_model_latest_infer(self):
-                                            "unexpected infer stats for v" + str(v) + " model " + model_name)
+                            if pair[1] == "http":
+                                stats = infer_stats['model_stats'][0]['inference_stats']
+                                self.assertEqual(stats['success']['count'], 0,
+                                                 "unexpected infer stats for version " + str(v) +
+                                                  " of model " + model_name)
+                            else:
+                                stats = infer_stats.model_stats[0].inference_stats
+                                self.assertEqual(stats.success.count, 0,
+                                                 "unexpected infer stats for version " + str(v) +
+                                                  " of model " + model_name)
 
             except InferenceServerException as ex:
                 self.assertTrue(False, "unexpected error {}".format(ex))
 
     def test_model_specific_infer(self):
         input_size = 16
+        tensor_shape = (1, input_size)
 
         # There are 3 versions of *_float32_float32_float32 but only
         # versions 1 and 3 should be available.
         for platform in ('graphdef', 'netdef', 'plan'):
-            tensor_shape = (input_size,)
+            tensor_shape = (1, input_size)
             model_name = platform + "_float32_float32_float32"
 
             # Initially there should be no version status...
             try:
-                for pair in [("localhost:8000", ProtocolType.HTTP), ("localhost:8001", ProtocolType.GRPC)]:
-                    server_status0, req_id0 = _get_server_status(pair[0], pair[1], model_name)
-                    self.assertTrue(model_name in server_status0.model_status,
-                                    "expected status for model " + model_name)
-                    self.assertEqual(len(server_status0.model_status[model_name].version_status), 2,
-                                    "expected status for 2 versions for model " + model_name)
+                for pair in [("localhost:8000", "http"), ("localhost:8001", "grpc")]:
+                    if pair[1] == "http":
+                        triton_client = httpclient.InferenceServerClient(
+                            url=pair[0], verbose=True)
+                    else:
+                        triton_client = grpcclient.InferenceServerClient(
+                            url=pair[0], verbose=True)
+
+                    self.assertTrue(triton_client.is_server_live())
+                    self.assertTrue(triton_client.is_server_ready())
+                    model_metadata = triton_client.get_model_metadata(model_name)
+
+                    if pair[1] == "http":
+                        self.assertEqual(model_name, model_metadata['name'])
+                    else:
+                        self.assertEqual(model_name, model_metadata.name)
+    
                     for v in (1, 3):
```

Comment:
Also test that version 2 is not ready. Don't need a loop just 3 calls to is_model_ready